src/serializers/snowflake.ts

```diff
@@ -16,8 +16,9 @@ export default class extends Serializer {
 
 	public validate(data: string, { entry, language }: SerializerUpdateContext) {
 		if (this.kRegExp.test(data)) {
-			const snowflake = new Snowflake(data);
-			if (snowflake.timestamp >= this.kMinimum && snowflake.timestamp < Date.now()) return data;
+			const snowflake = DiscordSnowflake.deconstruct(data);
+			const timestamp = Number(snowflake.timestamp);
+			if (timestamp >= this.kMinimum && timestamp < Date.now()) return data;
```

Comment:
nit: why not <=?

Reply:
Because a message that was sent the same millisecond Skyra received it, is practically impossible, we'd talk about sub-millisecond round-trips.